### DISCUSSION TITLE
How to make a pull request from branch a, to branch b

### DISCUSSION THREAD

**Discussion by CondratKenasil** [community/community#39864] [New to GitHub] [answered]
### Select Topic Area

Question

### Body

Hi, I am new to github, can someone tell me how to pull?

**Comment by MitkoVtori** [comment, marked as answer]
Go to the repo, under the repo name click pull request and then new pull request.
![Screenshot 2022-11-23 201335](https://user-images.githubusercontent.com/112943652/203619521-f2c922c6-60ac-4c34-815c-22261b4e919d.png)
Then select from which branch to branch and you're ready :)
![Screenshot 2022-11-23 201409](https://user-images.githubusercontent.com/112943652/203619638-69de303a-87f9-4311-9360-949311611138.png)

**Reply by CondratKenasil** [reply]
Thank you! It worked.

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @CondratKenasil , we're happy you're here! I've gone ahead and moved this discussion to **New to GitHub**. There is a lot of good resources to help. Also check out [GitHub Skills](https://skills.github.com/). Good luck!
